### ISSUE TITLE
Catch "server is current unable to handle the request" errors and print something nicer

### ISSUE THREAD

**Issue by wwitzel3** [vmware-tanzu/octant#55]
@bryanl commented on [Tue Apr 16 2019](https://github.com/heptio/developer-dash/issues/731)

Error message looks like:

```
2019-04-16T08:12:55.021-0400    ERROR   api/stream.go:255       event generator error: adding tab: unable to retrieve the complete list of server APIs: metrics.k8s.io/v1beta1: the server is currently unable to handle the request
github.com/heptio/developer-dash/internal/api.(*contentStreamer).content.func2
        /Users/bryan/go/src/github.com/heptio/developer-dash/internal/api/stream.go:255
```

It should be possible to handle this error in a more graceful fashion

---

@wwitzel3 commented on [Wed Jun 05 2019](https://github.com/heptio/developer-dash/issues/731#issuecomment-499211597)

in the code that checks if a tab has an error, instead of skipping, add a component.Error to the tab.

---

@wwitzel3 commented on [Wed Jun 05 2019](https://github.com/heptio/developer-dash/issues/731#issuecomment-499231102)

related #794 

---

@wwitzel3 commented on [Wed Jun 05 2019](https://github.com/heptio/developer-dash/issues/731#issuecomment-499231294)

maybe have a list of errors in Config with a unread counter

**Comment by wwitzel3** [comment]
We will create a component for errors that can be displayed in the Octant Status View

**Comment by wwitzel3** [comment]
Recheck with new error handling code.

**Comment by wwitzel3** [comment]
resolved by #544
